src/test/suite/fixtures/recorded/hatTokenMap/bringPointAndHarpToEndOfSecondCarWhaleAndEndOfJustWhaleTakeWhale.yml

```diff
@@ -13,15 +12,22 @@ command:
     - type: list
       elements:
         - type: primitive
-          position: after
-          insideOutsideType: inside
-          selectionType: token
-          modifier: {type: subpiece, pieceType: character, anchor: 1, active: 1, excludeAnchor: false, excludeActive: false}
           mark: {type: decoratedSymbol, symbolColor: default, character: w}
+          modifiers:
+            - {type: position, position: end}
+            - type: ordinalRange
+              scopeType: {type: character}
+              anchor: 1
+              active: 1
+              excludeAnchor: false
+              excludeActive: false
         - type: primitive
-          position: after
-          insideOutsideType: inside
           mark: {type: decoratedSymbol, symbolColor: default, character: w}
+          modifiers:
+            - {type: position, position: end}
+            - {type: toRawSelection}
```

Comment:
Note that we add "just" here to break inference chain, because we actually had a bug where we weren't doing inference for subtoken modifiers that had a position, even though we should have been